### PR TITLE
Release notes 2.13

### PR DESCRIPTION
**Pull request by kvalliyurnatt**
Updating Release notes and Release highlights for the upcoming 2.13.0 release

### CODE REVIEW

File: docs/release-notes/2.13.0.asciidoc [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+== {n} version 2.13.0
+
+
+
+[[feature-2.13.0]]
+[float]
+=== New features
+
+* ECK Enterprise Search Helm Chart. {pull}7744[#7744]
+* feat(helm/operator): statefulset labels and annotations {pull}7688[#7688]
```
[the remark this answers is not in the document]
Comment (kvalliyurnatt) [reply docs/release-notes/2.13.0.asciidoc]: done

File: docs/release-notes/2.13.0.asciidoc [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+* fix(deps): update module github.com/hashicorp/vault/api to v1.12.2 {pull}7633[#7633]
+* fix(deps): update module github.com/google/go-containerregistry to v0.19.1 {pull}7632[#7632]
+* fix(deps): update k8s to v0.29.3 {pull}7631[#7631]
+* fix(deps): update module github.com/elastic/go-ucfg to v0.8.7 {pull}7610[#7610]
+* chore(deps): update registry.access.redhat.com/ubi9/ubi-minimal docker tag to v9.3-1612 {pull}7609[#7609]
+* chore(deps): update docker.io/library/golang docker tag to v1.22.1 {pull}7608[#7608]
+* Bump github.com/go-jose/go-jose/v3 from 3.0.1 to 3.0.3 {pull}7602[#7602]
+* fix(deps): update module golang.org/x/crypto to v0.21.0 {pull}7595[#7595]
+* fix(deps): update module github.com/stretchr/testify to v1.9.0 {pull}7593[#7593]
+* fix(deps): update module github.com/prometheus/client_golang to v1.19.0 {pull}7591[#7591]
```
Comment (thbkrkr) [inline docs/release-notes/2.13.0.asciidoc:71]: There are a few deps updates that we can remove so we only keep the latest one. I added the `exclude-from-release-notes` label. Could you please regenerate?

```
* fix(deps): update module github.com/elastic/go-ucfg to v0.8.7 {pull}7610[#7610] # to exclude
* fix(deps): update module github.com/elastic/go-ucfg to v0.8.8 {pull}7641[#7641]
* fix(deps): update module github.com/hashicorp/vault/api to v1.12.2 {pull}7633[#7633] # to exclude
* fix(deps): update module github.com/hashicorp/vault/api to v1.13.0 {pull}7734[#7734]
* fix(deps): update module github.com/prometheus/common to v0.52.2 {pull}7695[#7695] # to exclude
* fix(deps): update module github.com/prometheus/common to v0.53.0 {pull}7735[#7735]
* fix(deps): update module golang.org/x/crypto to v0.21.0 {pull}7595[#7595] # to exclude
* fix(deps): update module golang.org/x/crypto to v0.22.0 {pull}7696[#7696]
```

Comment (kvalliyurnatt) [reply docs/release-notes/2.13.0.asciidoc:67]: done. Please take another look to make sure this looks okay.

**Review by thbkrkr — APPROVED** [review]

**Comment by kvalliyurnatt** [comment]
## 💚 All backports created successfully

| Status | Branch | Result |
|:------:|:------:|:------|
|✅|2.13|[](https://github.com/elastic/cloud-on-k8s/pull/7809)|

### Questions ?
Please refer to the [Backport tool documentation](https://github.com/sqren/backport)

**Pull request merged**